cmd/manager/v1alpha3/main.go

```diff
@@ -194,8 +196,11 @@ func (s *server) Check(ctx context.Context, in *health_pb.HealthCheckRequest) (*
 func main() {
 	flag.Parse()
 	var err error
-
-	dbIf, err = kdb.New()
+	dbName := os.Getenv("DB_NAME")
```

Comment:
maybe we can add pkg/db/v1alpha3/common/const.go to define all costants including ENV name, db backend name and so on.

Reply:
Done